modules/op-release-notes-1-4.adoc

```diff
@@ -124,6 +129,8 @@ fsGroup:
 
 * On IBM Power Systems, IBM Z, and LinuxONE, the `tkn hub` command is not supported.
 
+* On X86 architectures, IBM Power Systems, IBM Z, and LinuxONE, a Jib Maven cluster task fails when the docker pull rate limit is exceeded.
```

Comment:
I don't think that we need to mention this in RN. This applies to all tasks pulling images from Docker Hub (only Docker Hub, other image registries are not affected).